### PR TITLE
Added buffer between Lifecycle and private Label

### PR DESCRIPTION
**Pull request by snittoor**
Added space between labels showing Lifecycle status of Api and Public/private status of a Api in Gird/Table view. Closes #223 .

### CODE REVIEW

**Comment by marla-singer** [comment]
@snittoor You should connect your PR and issue via phrase "Closes #" and issue number in description box. In your case you need to write **"Closes #223"** (no bold)

File: api_catalog/client/table/table.html [hunk 1]
```diff
@@ -18,11 +18,13 @@ <h4 class="api-catalogue-table-title">
...
             <!-- Api LifeCycleStatus Label -->
-            {{> apiLifecycleStatus api=api }}
-            
-            {{# unless api.isPublic }}
-              <span class="label label-default label-align">private</span>
-            {{/ unless }}
+                {{> apiLifecycleStatus api=api }}
+               &nbsp;
+                  {{# unless api.isPublic }}
+                         <span  class="label label-default label-align">private</span>
```
Comment (marla-singer) [inline api_catalog/client/table/table.html]: Please refactor this part. Indent file with two spaces for each level of nesting.

Comment (snittoor) [inline api_catalog/client/table/table.html]: Sure , will keep in mind next time and indent code accordingly.I can make the changes if you want.

Comment (marla-singer) [inline api_catalog/client/table/table.html]: Yes, please, make the changes.

**Pull request closed without merging**